src/Nox.Types/Types/HashedText/HashedText.cs

```diff
@@ -1,9 +1,113 @@
+using System;
+using System.Security.Cryptography;
+using System.Text;
+
 namespace Nox.Types;
 
+/// <summary>
+/// Represents a Nox <see cref="HashedText"/> type and value object.
+/// </summary>
+public sealed class HashedText : ValueObject<(string HashText, string Salt), HashedText>
+{
+    public string HashText
+    {
+        get => Value.HashText;
+        private set => Value = (HashText: value, Value.Salt);
+    }
+    public string Salt
+    {
+        get => Value.Salt;
+        private set => Value = (Value.HashText, Salt: value);
+    }
+
+    public HashedText() { Value = (string.Empty, string.Empty); }
+
+    public static HashedText From(string value, HashedTextTypeOptions options)
```

Comment:
Can we have XML documentation on the public methods at least?

Reply:
Documentation added.